### PR TITLE
gdal: add with_python_bindings option

### PR DESCRIPTION
**Pull request by marxin**

### CODE REVIEW

File: recipes/gdal/post_3.5.0/conandata.yml [hunk 1]
```diff
@@ -12,9 +12,15 @@ patches:
   "3.7.0":
     - patch_file: "patches/3.7.0/0-replace-find-package.patch"
       base_path: "source_subfolder"
+    - patch_file: "patches/3.7.0/1-python-binding-find-libdir.patch"
+      base_path: "source_subfolder"
```
Comment (valgur) [inline recipes/gdal/post_3.5.0/conandata.yml:16]: Could you add a patch_description info to these patches, please?

Comment (marxin) [reply recipes/gdal/post_3.5.0/conandata.yml:16]: Sure, done.

**Review by valgur — APPROVED** [review]
Thanks! LGTM

**Review by AndreyMlashkin — APPROVED** [review]

**Comment by AndreyMlashkin** [comment]
@SSE4 @uilianries @SpaceIm mb reopen this one?

**Review by SSE4 — APPROVED** [review]

**Comment by SSE4** [comment]
> @SSE4 @uilianries @SpaceIm mb reopen this one?

I don't have a permission to re-open.

**Pull request closed without merging**